Tests/AWSSDKSwiftCoreTests/QueryEncoderTests.swift

```diff
@@ -13,7 +13,7 @@ class QueryEncoderTests: XCTestCase {
 
     func queryString(dictionary:[String:Any]) -> String? {
         var components = URLComponents()
-        components.queryItems = dictionary.map( {URLQueryItem(name:$0.key, value:String(describing: $0.value))} ).sorted(by: { $0.name < $1.name })
+        components.queryItems = dictionary.map( {URLQueryItem(name:$0.key, value:String(describing: $0.value).addingPercentEncoding(withAllowedCharacters: AWSClient.queryAllowedCharacters)!)} ).sorted(by: { $0.name < $1.name })
```

Comment:
Why have you added this line. You are percent encoding the text twice. `URLComponents` percent encodes the url already.

Reply:
Oh, it's my mistake, I have removed it.